domain-ee/ee-ep-merge-app/src/python_src/model/merge_job.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+from typing import Any
+from uuid import UUID
+
+from pydantic import BaseModel, conint
+
+
+class JobState(str, Enum):
+    COMPLETED_ERROR = 'COMPLETED_ERROR'
+    PENDING = 'PENDING'
+    RUNNING_GET_PENDING_CLAIM_CONTENTIONS = 'RUNNING_GET_PENDING_CLAIM_CONTENTIONS'
+    RUNNING_GET_SUPP_CLAIM_CONTENTIONS = 'RUNNING_GET_SUPP_CLAIM_CONTENTIONS'
+    RUNNING_SET_TEMP_STATION_OF_JURISDICTION = 'RUNNING_SET_TEMP_STATION_OF_JURISDICTION'
+    RUNNING_MERGE_CONTENTIONS = 'RUNNING_MERGE_CONTENTIONS'
+    RUNNING_UPDATE_PENDING_CLAIM_CONTENTIONS = 'RUNNING_UPDATE_PENDING_CLAIM_CONTENTIONS'
+    RUNNING_CANCEL_SUPP_CLAIM = 'RUNNING_CANCEL_SUPP_CLAIM'
+    COMPLETED_SUCCESS = 'COMPLETED_SUCCESS'
+
+    def __str__(self):
+        return self.value
```

Comment:
Same comment as I made on the other PR about using the more idiomatically Pythonic way to define enums.